defaults/main.yml

```diff
@@ -33,6 +47,19 @@ OPENCRAFT_ENV_TOKENS:
   SECRET_KEY: 'your-random-secret-key'
   WATCH_ORGANIZATION: 'your-github-organization'
 
+# These env tokes are generated from other ansible variables, entries in
+# this dictionary can be overriden by OPENCRAFT_ENV_TOKENS
+OPENCRAFT_GENERATED_ENV_TOKENS:
```

Comment:
Ah cool that makes sense.. I didn't realize that `OPENCRAFT_ENV_TOKENS` gets overridden for each deployment.

Thank you for the explanation! :)